README.md

```diff
@@ -4,6 +4,123 @@ Context Maps describe the contact between bounded contexts and teams with a coll
+
+The team providing an Open-host Service is usually in an upstream position whereas the clients using it are downstream teams. The teams on the downstream are free to be conformists or to build anticorruption layers.
+
+#### Conformist
+
+<img src="resources/cf.jpg" alt="Conformist" height=150/>
+
+"Eliminate the complexity of translation between bounded contexts by slavishly adhering to the model of the upstream team. Although this cramps the style of the downstream designers and probably does not yield the ideal model for the application, choosing conformity enormously simplifies integration. Also, you will share a ubiquitous language with your upstream team. The upstream is in the driver’s seat, so it is good to make communication easy for them. Altruism may be sufficient to get them to share information with you." ([Source: DDD Reference by Eric Evans](https://www.domainlanguage.com/ddd/reference/))
+
+#### Anticorruption Layer
+
+<img src="resources/acl.jpg" alt="Anticorruption Layer" height=150/>
+
+"As a downstream client, create an isolating layer to provide your system with functionality of the upstream system in terms of your own domain model. This layer talks to the other system through its existing interface, requiring little or no modification to the other system. Internally, the layer translates in one or both directions as necessary between the two models." ([Source: DDD Reference by Eric Evans](https://www.domainlanguage.com/ddd/reference/))
+
+#### Shared Kernel
+
+<img src="resources/shared-kernel.jpg" alt="Shared Kernel" width=300/>
+
+"Designate with an explicit boundary some subset of the domain model that the teams agree to share. Keep this kernel small.Within this boundary, include, along with this subset of the model, the subset of code or of the database design associated with that part of the model. This explicitly shared stuff has special status, and shouldn’t be changed without consultation with the other team." ([Source: DDD Reference by Eric Evans](https://www.domainlanguage.com/ddd/reference/))
+
+#### Partnership
+
+<img src="resources/partnership.jpg" alt="Partnership" width=300/>
+
+"Where development failure in either of two contexts would result in delivery failure for both, forge a partnership between the teams in charge of the two contexts. Institute a process for coordinated planning of development and joint management of integration. 
+The teams must cooperate on the evolution of their interfaces to accommodate the development needs of both systems. Interdependent features should be scheduled so that they are completed for the same release." ([Source: DDD Reference by Eric Evans](https://www.domainlanguage.com/ddd/reference/))
+
```

Comment:
Interestingly I couldn't find 'Partnership' in the blue book. Could you? If it really isn't there, this relationship was names after 2004.

Reply:
It was added by Vaughn Vernon in the red book and later added to the DDD Reference by Eric Evans.

Reply:
That is interesting.